index.js

```diff
@@ -0,0 +1,63 @@
+import Books from './modules/books.js';
+/* eslint-disable no-unused-vars */
+import { navigateTo, menuItems } from './modules/forEachfunction.js';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] [OPTIONAL] It is not good practice to disable linters, as they are there to help us write better code.